geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl

```diff
@@ -0,0 +1,26 @@
+---
+# This file can be renamed as `docker-compose.override.yaml` and uncomment the desired lines for
+# development. The file `docker-compose.override.yaml` is ignored by Git by default.
+
+# yamllint disable rule:line-length
+
+version: '2'
+
+services:
+  geoportal:
+    volumes:
+      # For Python project development.
+      - ${PWD}/geoportal/{{package}}_geoportal:/app/{{package}}_geoportal
+      # For Python c2cgeportal development.
+      # - '${PWD}/../c2cgeoportal/commons/c2cgeoportal_commons:/opt/c2cgeoportal_commons/c2cgeoportal_commons'
+      # - '${PWD}/../c2cgeoportal/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal_geoportal/c2cgeoportal_geoportal'
+      # - '${PWD}/../c2cgeoportal/admin/c2cgeoportal_admin:/opt/c2cgeoportal_admin/c2cgeoportal_admin'
+    environment:
+      - GUNICORN_CMD_ARGS=--reload
+
+  # For Javascript project development.
+  # The debug application will be availavble at ``https://<host>/<entry_point>/dev/<interface>.html``.
+  webpack_dev_server:
+    extends:
+      file: docker-compose-lib.yaml
+      service: webpack_dev_server
```

Comment:
@sbrunner : Note that I had to remove geoportal entry point to use webpack_dev_server
```
geoportal:
    entrypoint: []
```
because of rights on ngeo tmpl files.
I do not know if this have been fixed.

Reply:
I just try again and fix the `webpack-dev-server`, but I didn't get any error without that..., and we need to interpret the tmpl files...

Reply:
Ok, I probably missed something, it will help anyway to create a development environment :-)